Sources/RichTextKit/Context/RichTextContext.swift

```diff
@@ -123,6 +122,12 @@ public class RichTextContext: ObservableObject {
     @Published
     public var isEditingText = false
 
+    /**
+    The size of the TextView
+     */
+    @Published
+    public var contentSize: CGSize = .zero
```

Comment:
Why is this published if you are not using any subscriber?

Reply:
https://github.com/danielsaidi/RichTextKit/issues/118